src/diffusers/pipelines/hunyuan_video/pipeline_hunyuan_video_image2video.py

```diff
@@ -782,10 +812,17 @@ def __call__(
             negative_prompt_attention_mask = negative_prompt_attention_mask.to(transformer_dtype)
             negative_pooled_prompt_embeds = negative_pooled_prompt_embeds.to(transformer_dtype)
 
-        # 4. Prepare timesteps
+        # 5. Prepare timesteps
         sigmas = np.linspace(1.0, 0.0, num_inference_steps + 1)[:-1] if sigmas is None else sigmas
         timesteps, num_inference_steps = retrieve_timesteps(self.scheduler, num_inference_steps, device, sigmas=sigmas)
 
+        # 6. Prepare guidance condition
+        guidance = None
```

Comment:
oh should we add a check now for true_cfg & guidance? I think we cannot use cfg with the guidance embeds, no?